hack/loadtest/util/placement.go

```diff
@@ -173,13 +183,10 @@ func collectApplyMetrics(ctx context.Context, hubClient client.Client, deadline,
 				LoadTestApplyLatencyMetric.WithLabelValues(currency, fleetSize).Observe(endTime.Seconds())
 				return fleetSize, clusterNames
 			} else if cond == nil || cond.Status == metav1.ConditionUnknown {
-				klog.V(5).Infof("the cluster resource placement `%s` is pending", crpName)
-			} else if cond != nil && condition.IsConditionStatusFalse(cond, 1) {
-				// failed
-				klog.V(2).Infof("the cluster resource placement `%s` failed", crpName)
-				LoadTestApplyCountMetric.WithLabelValues(currency, fleetSize, "failed").Inc()
-				applyFailCount.Inc()
-				return fleetSize, clusterNames
+				klog.V(2).Infof("the cluster resource placement `%s` is pending", crpName)
```

Comment:
Hi Britania! A nit: the code seems to be using leveled logs and unleveled logs in a mixed manner; is there any particular reason?

Reply:
I do not have a reason other than the original load test used both.